qiita_pet/handlers/admin_processing_job.py

```diff
@@ -109,21 +110,43 @@ def post(self):
         # Get user-inputted qiita id and sample names
         qid = self.get_argument("qid")
         snames = self.get_argument("snames").split()
-        error, matching, missing, extra, blank = [None]*5
+        error, matching, missing, extra, blank, duplicates = [None]*6
 
         # Stripping leading qiita id from sample names
         # Example: 1.SKB1.640202 -> SKB1.640202
         try:
-            qsnames = list(Study(qid).sample_template)
+            st = Study(qid).sample_template
```

Comment:
IMHO st should be changed to something more substantial. I made the same mistake thinking it was a tmp variable but it's actually used farther down in the code.